### DISCUSSION TITLE
Ability to select different rows

### DISCUSSION THREAD

**Discussion by danisal** [nick-keller/react-datasheet-grid#184] [Q&A] [answered]
Hi,

Is it possible to select different rows when clicking on the row number? for example, clicking on row number 4 will select also that row, and not row number 3

![image](https://user-images.githubusercontent.com/9869755/190226052-03c2215e-8229-4bb9-872e-13aaa2ba8c26.png)

**Comment by nick-keller** [comment, marked as answer]
Unfortunately no, multi selection is not supported sorry :/

**Comment by nick-keller** [comment]
But you could have a checkbox column to select multiple rows, and even add a colored background on selected rows for clarity.

**Reply by danisal** [reply]
Any CSS class that I should look for in order to color the entire row? when using a checkbox to mark the row?

**Reply by nick-keller** [reply]
this should work:
```css
.selected-row {
  --dsg-cell-background-color: #f00;
}
```

And then for each column add the [`.selected-row`](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#cellclassname) class when necessary:
```ts
const columns = [
  { cellClassName: ({ rowData }) => rowData.selected ? 'selected-row' : undefined }
]
```
